lib/generators/solid_queue/install/templates/db/queue_schema.rb

```diff
@@ -37,7 +37,9 @@
     t.string "concurrency_key"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "batch_id"
```

Comment:
I should add a foreign key here. Originally I was only creating batches if jobs actually enqueued, but now I always create them, and run an empty batch if no jobs end up enqueued.